client/src/components/relatedProducts/card.jsx

```diff
@@ -133,9 +133,10 @@ const Card = (props) => {
           <FontAwesomeIcon style={fAColor} icon={faCircleXmark} />
         }
 
-      </RelatedBtn>
+      </RelatedBtn >
 
       <div
+      data-testid = {props.btnStyle ==='x' ?  'outfit'+props.name : 'related'+ props.name}
         // onMouseEnter={hoverHandler}
```

Comment:
Good use of ternary operators here. It is always interesting reading other peoples' code because there are so many things I would have never though to do.

Reply:
I realize i didn't even end up using these yet. i should have commented it out. thanks